src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.component.html

```diff
@@ -11,7 +11,9 @@
                                 *ngIf="
                                     navItem.activeTab === settingsTabsEnum.SCHEDULING
                                         ? isSchedulingAvailable
-                                        : navItem.visible
+                                        : navItem.visible ||
+                                          (navItem.activeTab === settingsTabsEnum.COMPACTING &&
+                                              datasetBasics.kind === DatasetKind.Root)
```

Comment:
Move the logic to component and test it better. This is becoming complex.

Reply:
Moved logic to the component